core/src/poh_service.rs

```diff
@@ -24,6 +24,9 @@ pub const DEFAULT_PINNED_CPU_CORE: usize = 0;
 const TARGET_SLOT_ADJUSTMENT_NS: u64 = 50_000_000;
 
 impl PohService {
+    // Note that poh_config is a snapshot at this function call.
+    // At runtime, the rest of the system may transition to use a new poh_config instance.
```

Comment:
Think we should consider just removing the Arc and passing cloned versions to these services. The Arc makes it look like it's shared and somewhat immutable, but it's not really the case. May obviate the need for this comment.